File: java/client/src/org/openqa/selenium/devtools/network/types/SignedExchangeError.java
package org.openqa.selenium.devtools.network.types;

import org.openqa.selenium.json.JsonInput;

/**
 * Information about a signed exchange response
 */
public class SignedExchangeError {

  private String message;

  private Integer signatureIndex;

  private SignedExchangeErrorField errorField;

  /** Error message. */
  public String getMessage() {
    return message;
  }

  /** Error message. */
  public void setMessage(String message) {
    this.message = message;
  }

  /** The index of the signature which caused the error. */
  public Integer getSignatureIndex() {
    return signatureIndex;
  }

  /** The index of the signature which caused the error. */
  public void setSignatureIndex(Integer signatureIndex) {
    this.signatureIndex = signatureIndex;
  }

  /** The field which caused the error. */
  public SignedExchangeErrorField getErrorField() {
    return errorField;
  }

  /** The field which caused the error. */
  public void setErrorField(SignedExchangeErrorField errorField) {
    this.errorField = errorField;
  }

  public static SignedExchangeError parseResponse(JsonInput input) {
    //TODO: @GED implement parser
    return null;
  }
}

File: dotnet/src/webdriver/DevTools/AutoGenerated/Network/SignedExchangeError.cs
namespace OpenQA.Selenium.DevTools.Network
{
    using Newtonsoft.Json;

    /// <summary>
    /// Information about a signed exchange response.
    /// </summary>
    public sealed class SignedExchangeError
    {
        /// <summary>
        /// Error message.
        ///</summary>
        [JsonProperty("message")]
        public string Message
        {
            get;
            set;
        }
        /// <summary>
        /// The index of the signature which caused the error.
        ///</summary>
        [JsonProperty("signatureIndex", DefaultValueHandling = DefaultValueHandling.Ignore)]
        public long? SignatureIndex
        {
            get;
            set;
        }
        /// <summary>
        /// The field which caused the error.
        ///</summary>
        [JsonProperty("errorField", DefaultValueHandling = DefaultValueHandling.Ignore)]
        public SignedExchangeErrorField? ErrorField
        {
            get;
            set;
        }
    }
}